packages/strapi/lib/middlewares/public/index.js

```diff
@@ -116,7 +106,7 @@ module.exports = strapi => {
           await next();
         },
         koaStatic(buildDir, {
-          index: 'index.html',
+          index: indexFileName,
           maxage: maxAge,
           defer: false,
         })
```

Comment:
At line 117, it uses `/index.html`, but I don't understand what it does. I think we need to handle this in case `index.html `doesn't exist. Anyone knows ?

Reply:
(lines are bellow, hidden)

Reply:
the index option tel koaStatic to serve the 'index.html' file in the folder  when the / route is called if it exists.

Reply:
I'm sorry, I wasn't precise, I was talking about a line that is after in the code (you have to expand). I couldn't comment on the line because I haven't modified it.
Can you check again please? :)